prototypes/styletransfer/model.py

```diff
@@ -116,15 +116,17 @@ def run_style_transfer(
             layer.trainable = False
 
         # Get the style and content feature representations (from our specified intermediate layers)
-        style_features, content_features = self.feature_representations(
+        content_features, style_features = self.feature_representations(
             content_img, style_img
         )
         gram_style_features = [
             gram_matrix(style_feature) for style_feature in style_features
         ]
 
         # Set initial image
-        init_image = tfe.Variable(content_img, dtype=tf.float32)
+        # TODO(glanaro): is it correct to preprocess or we don't really care?
+        init_img = self._process_img(content_img)
```

Comment:
Using `content_img` to initialize `init_img` does not work well in practice, in my experiments, as the `content` component of the loss is, by definition, counted twice. Starting from a random noise image yields more visually appealing results, with a stronger `style` component. We need to explore this further.

Reply:
```
def run_style_transfer(
        self,
        content_img: np.array,
        style_img: np.array,
        num_iterations=1000,
        content_weight=1e3,
        style_weight=1e-2,
    ):
```
`content_weight` and `style_weight` need to be calculated on the fly as they vary according to the image. We need to adjust the weights so that content and style don't show orders of magnitude of difference (which is the default case). To give an example, in my experiments, I was often obliged to set `content_weight=1000` and `style_weight=1`. Ideally we need to run a couple of optimization iterations and compute the content2style ratio, then use it to execute the actual style transfer process. As soon as the 2 losses are comparable, then we can add a custom coefficient to let the end user decide which proportion of content/style he desires.

Reply:
For the content2style ratio I created a new issue because it's outside of the scope of this PR, for the image initialization I'll add both versions (random or content) as a configuration option.